source-hubspot-native/test.flow.yaml

```diff
@@ -7,35 +7,118 @@ captures:
       - resource:
           name: contacts
+          interval: PT30S
         target: acmeCo/contacts
       - resource:
           name: deals
+          interval: PT30S
         target: acmeCo/deals
       - resource:
           name: engagements
+          interval: PT30S
         target: acmeCo/engagements
+      - resource:
+          name: contact_lists
+          interval: PT60S
+        target: acmeCo/contact_lists
+      - resource:
+          name: contacts_lists_subscription
+          interval: PT60S
+        target: acmeCo/contacts_lists_subscription
+      - resource:
+          name: campaigns
+          interval: PT60S
+        target: acmeCo/campaigns
+      - resource:
+          name: subscription_changes
+          interval: PT60S
+        target: acmeCo/subscription_changes
+      - resource:
+          name: email_events
+          interval: PT60S
+        target: acmeCo/email_events
+      - resource:
+          name: ticket_pipelines
+          interval: PT60S
+        target: acmeCo/ticket_pipelines
+      - resource:
+          name: deal_pipelines
+          interval: PT60S
+        target: acmeCo/deal_pipelines
+      - resource:
+          name: engagements_calls
+          interval: PT30S
+        target: acmeCo/engagements_calls
+      - resource:
+          name: engagements_emails
+          interval: PT30S
+        target: acmeCo/engagements_emails
+      - resource:
+          name: engagements_meetings
+          interval: PT30S
+        target: acmeCo/engagements_meetings
+      - resource:
+          name: engagements_notes
+          interval: PT30S
+        target: acmeCo/engagements_notes
+      - resource:
+          name: engagements_tasks
+          interval: PT30S
+        target: acmeCo/engagements_tasks
+      - resource:
+          name: goal_targets
+          interval: PT30S
+        target: acmeCo/goal_targets
+      - resource:
+          name: feedback_submissions
+          interval: PT30S
+        target: acmeCo/feedback_submissions
+      - resource:
+          name: line_items
+          interval: PT30S
+        target: acmeCo/line_items
+      - resource:
+          name: products
+          interval: PT30S
+        target: acmeCo/products
       - resource:
           name: tickets
+          interval: PT30S
         target: acmeCo/tickets
+      - resource:
+          name: marketing_emails
+          interval: PT30S
+        target: acmeCo/marketing_emails
+      - resource:
+          name: email_subscriptions
+          interval: PT30S
+        target: acmeCo/email_subscriptions
+      - resource:
+          name: marketing_forms
+          interval: PT30S
+        target: acmeCo/marketing_forms
+      - resource:
+          name: owners
+          interval: PT30S
+        target: acmeCo/owners
+      - resource:
+          name: workflows
+          interval: PT30S
+        target: acmeCo/workflows
       - resource:
           name: properties
-          interval: P1D
```

Comment:
Just spot checking: The table in the PR description says that this is supposed to be 1 day. But its 30 seconds here. Which one is it supposed to be?

Reply:
Updated at ae65e51c85616092acd08ece0c3ace515985e4d3